### PR TITLE
Add user-assigned identity sample.

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
Sample includes creating a user-assigned identity, then adding and
removing one from a VM.

Thanks for your contribution! Please let us know what problems this PR addresses or what functionality it adds. Please link to other related issues and PRs to speed up review. Thank you!

### CODE REVIEW

File: compute/vm_with_identity.go [hunk 1]
```diff
@@ -10,6 +10,8 @@ import (
 	"fmt"
 
 	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-06-01/compute"
+	"github.com/Azure/azure-sdk-for-go/services/preview/msi/mgmt/2015-08-31-preview/msi"
+	"github.com/pkg/errors"
```
Comment (vladbarosan) [inline compute/vm_with_identity.go]: dont we need this in the toml file ?

Comment (jhendrixMSFT) [reply compute/vm_with_identity.go]: This is part of the standard library.  It got pulled in from calls to ```errors.Wrap()```.  See [this](https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully) for more context.

Comment (vladbarosan) [reply compute/vm_with_identity.go]: is it ? https://godoc.org/github.com/pkg/errors vs package errors ?

Comment (jhendrixMSFT) [reply compute/vm_with_identity.go]: I stand corrected.  I'll run ```dep ensure -update``` and push the changes.

**Comment by grumblegirl** [comment]
Hi @jhendrixMSFT , are there sample golang project for pacakge resourcegraph from azure-sdk-for-go? (https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/services/resourcegraph/mgmt/2019-04-01/resourcegraph?tab=overview)

**Comment by jhendrixMSFT** [comment]
Hello @grumblegirl, unfortunately there isn't a sample for that package at present.  Please feel free to open an issue to track this request however I don't know when we'll be able to get to it.
CC @ArcturusZhang

**Pull request merged**